ocs_ci/helpers/storage_agnostic_dr_helpers.py

```
@@ -0,0 +1,866 @@
+        )
+
+        run_cmd(
+            f"oc set image deployment/mock-storage-operator-controller-manager"
+            f" manager={constants.MOCK_STORAGE_OPERATOR_IMAGE}"
+            f" -n {constants.MOCK_STORAGE_OPERATOR_NAMESPACE}",
+            cluster_config=cluster,
+        )
+
+        logger.info(
+            f"Waiting for mock storage operator pod in namespace"
+            f" '{constants.MOCK_STORAGE_OPERATOR_NAMESPACE}'"
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)